app/assets/javascripts/geoblacklight/viewers/map.js

```diff
@@ -9,14 +9,20 @@ GeoBlacklight.Viewer.Map = GeoBlacklight.Viewer.extend({
     bbox: [[-80, -195], [80, 185]],
     opacity: 0.75
   },
-  
+
   overlay: L.layerGroup(),
 
   load: function() {
     if (this.data.mapBbox) {
       this.options.bbox = L.bboxToBounds(this.data.mapBbox);
     }
     this.map = L.map(this.element).fitBounds(this.options.bbox);
+
+    // Add initial bbox to map element for easier testing
```

Comment:
I'm a little unsure here why we need to set this. Shouldn't this be coming from the view already?